Source/WebCore/dom/Document.cpp

```diff
@@ -403,7 +404,10 @@ static void CallbackForContainIntrinsicSize(const Vector<Ref<ResizeObserverEntry
             ASSERT(box->style().hasAutoLengthContainIntrinsicSize());
 
             auto contentBoxSize = entry->contentBoxSize().at(0);
-            target->setLastRememberedSize(ResizeObserverSize::create(contentBoxSize->inlineSize(), contentBoxSize->blockSize()));
+            if (box->style().containIntrinsicWidthType() == ContainIntrinsicSizeType::AutoAndLength)
+                box->isHorizontalWritingMode() ? target->setLastRememberedLogicalWidth(LayoutUnit(contentBoxSize->inlineSize())) : target->setLastRememberedLogicalHeight(LayoutUnit(contentBoxSize->blockSize()));
```

Comment:
It's better to use `if/else` than the trigraph operator for code like this.

Reply:
Done